README.md

```diff
@@ -13,7 +13,7 @@ on request.
 
 The discovery and presention of software package license information is a complex
 problem facing organizations that rely on open source software within their 
-innovation streams. dosocs2 enables easy creation of a "bill of materials" for any 
+innovation streams. dosocs2 enables creation of an SPDX document for any 
 software package to represent associated license information. In addition, dosocs2 and 
```

Comment:
@mlinksva The discovery of software package license information and the presentation of software package license information may have been the intent here. 

I may be wrong about it. @germonprez please do add in your thoughts.